samcli/commands/_utils/options.py

```diff
@@ -551,8 +551,10 @@ def manifest_option(f):
 
 def cached_click_option():
     return click.option(
-        "--cached",
+        "--cached/--no-cached",
```

Comment:
Can we add some tests to make sure `--no-cached` is working fine?

Reply:
Working on adding integration tests